src/bucket/test/BucketManagerTests.cpp

```diff
@@ -538,6 +538,7 @@ TEST_CASE("bucketmanager do not leak empty-merge futures",
         static_cast<uint32_t>(
             Bucket::FIRST_PROTOCOL_SUPPORTING_INITENTRY_AND_METAENTRY) -
         1;
+    cfg.TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION = cfg.LEDGER_PROTOCOL_VERSION;
```

Comment:
There are a few places where you update both `LEDGER_PROTOCOL_VERSION` and `TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION` to the same value. I would add a function in `Config` for this.

Reply:
I've checked again, in most cases it is sufficient to update either `TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION ` or `LEDGER_PROTOCOL_VERSION` (both are not required to be set together). Here since `cfg.USE_CONFIG_FOR_GENESIS` was set to true (in `getTestConfig`), setting `TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION` is the intention here and is sufficient.

Reply:
I'm not really sure this is what we want? I think the goal here should be to keep the behavior of the existing test cases the same. By setting `TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION` to a lower version and keeping `LEDGER_PROTOCOL_VERSION` the same, you create this range that technically isn't the same as before. What do you think about this? Does this actually matter?